### ISSUE TITLE
Problem with How to add New Models Documentation

### ISSUE THREAD

**Issue by yenhochen** [Nixtla/neuralforecast#1078]
### Description

Hi. there is a bug in the [**"How to add new Models to NeuralForecast"**](https://nixtlaverse.nixtla.io/neuralforecast/examples/how_to_add_models.html) tutorial. Specifically, in section 2.a where the model class is defined, the following code produces an error.

```
    def forward(self, windows_batch): # <<--- Receives windows_batch dictionary
        # Parse windows_batch
        insample_y = windows_batch['insample_y'].clone()
        # MLP
        y_pred = self.mlp(y_pred)
        # Reshape and map to loss domain
        y_pred = y_pred.reshape(batch_size, self.h, self.loss.outputsize_multiplier)
        y_pred = self.loss.domain_map(y_pred)
        return y_pred
```

This is caused specfically by the line `y_pred = self.mlp(y_pred)` because y_pred is not defined anywhere previously.

### Link

https://nixtlaverse.nixtla.io/neuralforecast/examples/how_to_add_models.html

**Comment by marcopeix** [comment]
You're right, it should be `y_pred = self.mlp(insample_y)`.

**Comment by elephaint** [comment]
Small fix for this included in #1090
